lua/telescope/builtin/__files.lua

```diff
@@ -538,13 +540,23 @@ files.current_buffer_fuzzy_find = function(opts)
       previewer = conf.grep_previewer(opts),
       attach_mappings = function()
         action_set.select:enhance {
+          pre = function(bufnr)
+            local selection = action_state.get_selected_entry()
+            if not selection then
+              return
+            end
+            local current_picker = action_state.get_current_picker(bufnr)
+            local searched_for = require("telescope.actions.state").get_current_line()
+            local highlighted = current_picker.sorter:highlighter(searched_for, selection.ordinal)
+            column = highlighted[1]
```

Comment:
In my testing, it looks like the column numbers in `highlighted` isn't necessarily ordered. I think we need to get the smallest value more deterministically.

I think there's a off-by-one error as well. I think we need to `- 1` the value as done so in the highlighting step as well.
https://github.com/nvim-telescope/telescope.nvim/blob/0f865f17af4f9bc1587a0132414cdfd32d91852e/lua/telescope/pickers.lua#L483

Reply:
Interesting, during testing(see "after" in the PR description) it was jumping to the correct character. I will test again once I address the comments.